suresh/orderDetailsProject.js

```diff
@@ -0,0 +1,101 @@
+var items = {
```

Comment:
in Walmart best practice, constants are normally capitalized. This shows that the variable does not change.

var ITEMS = {}

Reply:
updated